### DISCUSSION TITLE
Borders of Cabinda

### DISCUSSION THREAD

**Discussion by HeidiGoes** [community/community#21443] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, this is my first topic here… I have been using QGIS for some years now, to make maps for my PhD about Kikongo languages.<br>
I have a problem with the borders of “ne_10m_admin_0_countries” for Cabinda. The border is for a part equal with the river Chiloango, but on “ne_10m_admin_0_countries” it is a bit more to the north. Now I have done fieldwork there and noted several villages right there near the border. Now on the map it is as if they are in the DRC, but they are between the border and the river, where the border should be on the river.<br>
For just one or two villages I could change the coordinates a bit, but there are too many, and it would not show it right anyway, because the village now ‘almost on the border’ is not that close to the border at all. How can I change the border or could someone help to impove this part of the borders? Thanks<br>
Heidi (from Belgium)</p>

**Comment by Zerotask** [comment]
<p>This seems to be a very specific question and not really related to GitHub or Git.<br>
You should try to find a specific place for your question.</p>
<p>In general it’s also a good advice to post further information, e. g. a repository to look at your current code.</p>

**Comment by HeidiGoes** [comment]
<p>Thank you. I don’t make my own codes.<br>
Whom should I contact to get a better border layer?<br>
Or to know how to myself change the borders?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/heidigoes">@HeidiGoes</a> ,</p>
<p>GitHub is a platform where people collaborate to write code. You seem to have a specific question about a software product, which may or may not be developed over here at GitHub.</p>
<p>While all questions are welcome here in the Community, I’m afraid such a specific question won’t find an answer quickly.</p>
<p>If the product you’re trying to get help for has a presence on GitHub, you can <a href="https://github.com/search" rel="noopener nofollow ugc">search</a> for their repository on GitHub. Repositories have an “Issues” tab where questions specifically related to <strong>that</strong> project are discussed. You’ll probably have more succes there than this general forum.</p>
<p>Good luck!</p>
